esm-samples/README.md

```diff
@@ -1,12 +1,14 @@
-# @arcgis/core
+# @arcgis/core (ES modules)
 
-The projects in this directory are samples that integrate the [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) build of the ArcGIS API for JavaScript with various frameworks, module bundlers and build tools. 
+The sample projects in this directory integrate the [@arcgis/core](https://www.npmjs.com/package/@arcgis/core) build of the ArcGIS API for JavaScript with various frameworks, module bundlers and build tools. @arcgis/core is installed locally using NPM.
```

Comment:
@arcgis/core might look better as `@arcgis/core`

Reply:
Oh good catch, I'll fix that before merging.